### ISSUE TITLE
Readme example outdated: starbucks_uk

### ISSUE THREAD

**Issue by DieterDePaepe** [pbugnion/gmaps#258]
Loading the `starbucks_uk` dataset, mentioned in the readme, results in a KeyError.

`locations = gmaps.datasets.load_dataset("starbucks_uk")`

```
    120     Fetch a dataset, returning an array of tuples.
    121     """
--> 122     url = METADATA[dataset_name]["url"]
    123     column_types = METADATA[dataset_name]["types"]
    124     f = urlopen(url)

KeyError: 'starbucks_uk'
```

Using version 0.8.0, installed using the conda instructions in the Readme.

**Comment by pbugnion** [comment]
Ah good spot, thanks -- the example in the screenshot is correct, but the text is outdated.

I'll probably fix this this weekend, but feel free to submit a PR before that if you want!

**Comment by pbugnion** [comment]
Fixed by #260. Thanks for reporting.
